src/components/errors/ErrorNetworkMismatch.tsx

```diff
@@ -1,33 +1,35 @@
 import React from 'react'
 import { Button } from 'decentraland-ui/dist/components/Button/Button'
 import { ErrorContainer, ErrorDetails, ErrorImage } from './Error'
-import { reload } from "./utils"
-import { NETWORK } from '../../integration/url'
-
 import errorImage from '../../images/errors/robotsmiling.png'
+import { switchToChainId } from '../../eth/provider'
 import './errors.css'
+import { getChainName } from '@dcl/schemas/dist/dapps/chain-id'
```

Comment:
Maybe we want to move this to the top (below `'decentraland-ui/dist/components/Button/Button'`) to keep consistent with other file's order